xdsl/dialects/csl/csl.py

```diff
@@ -1445,6 +1446,29 @@ class SymbolExportOp(IRDLOperation):
 
     type = prop_def(base(PtrType) | base(FunctionType))
 
+    def __init__(self, sym_name: str | StringAttr, type_or_op: SSAValue | FunctionType):
+        """
+        Accepts one of:
+          1. var_name + PtrType + operand
+          2. var_name + FunctionType
```

Comment:
This was a bit confusing, as the function accepts 2 arguments in each case, but the second argument can be either an ssa value or a function type

Reply:
Sorry, this should be what the class accepts. The init simplifies this.

Reply:
Updated